### PR TITLE
More compara recipes

### PR DESCRIPTION
**Pull request by muffato**
Example output for the synthelogs
```
oryza_sativa    Os02t0776900-01 oryza_rufipogon ORUFI02G36060.1 100
oryza_sativa    Os02t0776900-01 oryza_nivara    ONIVA02G36590.1 100
oryza_sativa    Os02t0776900-01 oryza_indica    BGIOSGA005497-PA        100
oryza_sativa    Os02t0776900-01 oryza_glaberrima        ORGLA02G0298900.1       100
oryza_sativa    Os02t0776900-01 oryza_barthii   OBART02G34550.1 100
oryza_sativa    Os02t0776900-01 oryza_glumipatula       OGLUM02G35690.1 100
oryza_sativa    Os02t0776900-01 oryza_meridionalis      OMERI02G32800.1 100
oryza_sativa    Os02t0776900-01 oryza_longistaminata    KN539071.1_FGP018       100
oryza_sativa    Os02t0776900-01 oryza_punctata  OPUNC02G31900.1 100
oryza_sativa    Os02t0776900-01 oryza_brachyantha       OB02G41370.1    100
```

### CODE REVIEW

File: demo_scripts/exampleAPI.pl [hunk 1]
```diff
@@ -195,3 +196,53 @@
...
+foreach my $homology (@homologies) {
+
+	# get one orthologue
+	my $prot = $homology->get_all_Members->[1];
+
+    # find all orthologues in rice
+    my @rice_homologies = @{$homology_adaptor->
+        fetch_all_by_Member($prot,
+            -METHOD_LINK_TYPE => 'ENSEMBL_ORTHOLOGUES',
+            -TARGET_TAXON => 'Oryza')};
```
Comment (muffato) [inline demo_scripts/exampleAPI.pl:231]: @brunocontrerasmoreira :Whilst writing this script, I remembered we have this `-TARGET_TAXON` filter, which you could use in the other example to get all the orthologues in _rosids_ ? I should have thought about this earlier ...

Comment (brunocontrerasmoreira) [reply demo_scripts/exampleAPI.pl:231]: Thanks, I'll add this to that recipe

**Comment by brunocontrerasmoreira** [comment]
I leave user 'anonymous' as it works in my setting and it's the one suggested http://plants.ensembl.org/info/data/mysql.html

**Comment by muffato** [comment]
OK, thanks. I'll try to find out why it didn't work for me

**Pull request merged**